mesonbuild/mdist.py

```diff
@@ -70,7 +72,18 @@ def copy_git(src, distdir, revision='HEAD', prefix=None, subdir=None):
         t = tarfile.open(fileobj=f) # [ignore encoding]
         t.extractall(path=distdir)
 
-def process_submodules(src, distdir):
+msg_uncommitted_changes = 'Repository has uncommitted changes that will not be included in the dist tarball'
+msg_info_dirty_override = 'Use --allow-dirty to ignore the warning and proceed anyway'
```

Comment:
The second string is only used once, it should be inlined into the `mlog.error` down below.

The first string used to be duplicated in two places. I have no strong feelings about whether or not to move it into a variable.